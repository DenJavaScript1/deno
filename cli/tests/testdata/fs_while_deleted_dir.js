const pathname = await Deno.makeTempDir();
Deno.chdir(pathname);
await Deno.remove(pathname);
await Deno.mkdir("X", { recursive: true });
