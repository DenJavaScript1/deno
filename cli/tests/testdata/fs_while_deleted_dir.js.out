error: Uncaught (in promise) NotFound: Failed to get current working directory
await Deno.mkdir('X', { recursive: true });
^
    at async Object.mkdir (deno:runtime/js/30_fs.js:109:5)
    at async file://[WILDCARD]deno/cli/tests/testdata/fs_while_deleted_dir.js:4:1
